pkg/gateway/model/v3/model.go

```diff
@@ -12,6 +12,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
+const ChunkSize int = 64 * 1024
```

Comment:
Maybe name it `ChunkSizeBytes` so the units are clear?

Reply:
Or `MaxChunkBytes`

Reply:
Updated to MaxChunkSizeBytes